creator-node/scripts/run-tests.sh

```diff
@@ -29,7 +29,7 @@ if [ "${debug}" ]; then
   UNIT_TIMEOUT=0
   INTEGRATION_TIMEOUT=0
 else
-  UNIT_TIMEOUT=2000
+  UNIT_TIMEOUT=5000
```

Comment:
@SidSethi had to increase because the premium content middleware tests were taking longer and failing the 2000, even 3000 and sometimes 4000ms. probably because of the new proxyquire import. noticed that all other uses of proxyquire in tests are for integration tests in the test folder.
wondering if this test perhaps belongs there but not fully convinced?

Reply:
eh w/e - if this reliably passes with 5sec timeout, not worth much further investigation rn lets just ship